CRM/Contact/Form/CustomData.php

```diff
@@ -84,23 +84,18 @@ class CRM_Contact_Form_CustomData extends CRM_Core_Form {
    */
   public function preProcess() {
     $this->_cdType = $_GET['type'] ?? NULL;
-    $this->assign('cdType', FALSE);
+    $isBuildForm = ($_GET['type'] ?? NULL) && CRM_Utils_Request::retrieve('multiRecordDisplay', 'String', $this);
     $this->_multiRecordDisplay = CRM_Utils_Request::retrieve('multiRecordDisplay', 'String', $this);
-    if ($this->_cdType || $this->_multiRecordDisplay == 'single') {
-      if ($this->_cdType) {
-        $this->assign('cdType', TRUE);
-      }
+    $this->assign('cdType', CRM_Utils_Request::retrieve('multiRecordDisplay', 'String', $this));
```

Comment:
@eileenmcnaughton the most literal one-line version of all this would be
```suggestion
    $this->assign('cdType', (bool) $this->_cdType);
```
That would preserve the original logic exactly (which was doing it in a more roundabout way of assigning it to false then assigning it to true if $this->_cdType).
What you did is different than the original. I'm not sure exactly how this code works so I don't know if it matters. But it's different.